momepy/graph.py

```diff
@@ -1031,7 +1031,7 @@ def node_density(
 
     Examples
     --------
-    >>> nodes['density'] = mm.node_density(nodes, edges, graph)
+    >>> network_graph = mm.straightness_centrality(network_graph, radius=5)
```

Comment:
This also doesn't apply :D

Reply:
i changed the radius, but not the name....